src/models/Scholarships.ts

```diff
@@ -15,6 +16,8 @@ interface ScholarshipData {
   year?: string;
   authorId?: string;
   authorEmail?: string;
+  states?: String[];
+  eligibility?: ScholarshipEligibility;
```

Comment:
As `ScholarshipEligibility` isn't used elsewhere you could also define the interface here. E.g.
```ts
  eligibility?: {
    gpa?: number;
    ...
  };
```

I'm not strongly leaning one way or the other so feel free to keep it as is if you prefer.

Reply:
I'll leave it for now since we might use it elsewhere later.